package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@launchpadlab/lp-components",
-  "version": "8.0.0",
+  "version": "9.0.0",
```

Comment:
I figured I'd hold off on writing a migration guide until the code changes are approved.